tests/migration/reboot_to_migration.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Reboot machine to perform upgrade
+#       Just trigger reboot action, afterwards tests will be
+#       incepted by later test modules, such as tests in
+#       load_boot_tests or wait_boot in setup_zdup.pm
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
```

Comment:
Only include functions used in this module.

Reply:
OK.

Reply:
select_console,set_var, assert_screen include from testapi, you can double check others also.

Reply:
changed and verified https://openqa.suse.de/t12615294